### ISSUE TITLE
GET Paramater no class

### ISSUE THREAD

**Issue by thomasvanhesteren** [rnmp/salvattore#199]
When I load a page with a GET paramater like ?Test=Test Salvatore is not setting the column width class. For example the class size-1of2 is not set. The class stays empty...

When I load the page without the GET parameter, the class is set properly to size-1of2 (or any other size on page load)

So, when I use a GET parameter, the class of the column width is not set and stays empty. When I resize the browser and pass by the @media query widths than the class is set properly again.

In Short:
Adding a GET parameter to the page where I use Salvattore gives me an empty column width-class. When I resize, Salvattore does it's job properly.

Example:
With GET: `<div class>`
Without GET: `<div class='size-1of2'>`
